tests/functional/adapter/incremental/fixtures.py

```diff
@@ -75,6 +78,36 @@
         - name: color
 """
 
+tblproperties_a = """
```

Comment:
Tblproperties tests/fixtures looks nearly identical to Tags tests/fixtures because their syntax and behavior is currently the same.